tests/Tests/700__sandbox/701__perfromance_sandbox.robot

```diff
@@ -0,0 +1,81 @@
+*** Settings ***
+Documentation    Test Suite for Performance Sandbox Test cases
+Library         OperatingSystem
+Library         Collections
+Library         Process
+Library         String
+
+
+*** Variables ***
+${NAMESPACE}     openshift-kube-apiserver
+${LABEL_SELECTOR}     app=openshift-kube-apiserver
+${MEMORY_THRESHOLD}    102400
+${PERF_CODE}    go run setup/main.go --users 2000 --default 2000  --custom 0 --username "user" --workloads redhat-ods-operator:rhods-operator --workloads redhat-ods-applications:rhods-dashboard --workloads redhat-ods-operator:cloud-resource-operator --workloads redhat-ods-monitoring:blackbox-exporter --workloads redhat-ods-monitoring:grafana --workloads redhat-ods-monitoring:prometheus <<< y   #robocop:disable
+
+
+*** Test Cases ***
+Verify RHODS Performance For Sandbox Onboarding Process
+    [Documentation]    Performance test for RHODS operator onboard
+    ...    to Sandbox environment
+    [Tags]     ODS-1404
+    ...        Sandbox
+    ...        Performance-Test
+    Run    git clone https://github.com/codeready-toolchain/toolchain-e2e.git
+    Run Keyword And Continue On Failure    Run Performance Test On RHODS Operator
+    Verify Sandbox Toolchain Data
+
+
+*** Keywords ***
+Capture Memory Utilization Of Openshift API Server POD
+    [Documentation]  Capture and compare the realtime memory utilization
+    ${memory_usage}     Run    kubectl top pod -n ${NAMESPACE} -l ${LABEL_SELECTOR} | awk '{if(NR>1)print $3}'
+    ${memory}    Split String    ${memory_usage}   \n
+    IF   len(${memory}) < ${3}
+         FAIL     Check if all the pods are available
+    ELSE
+        @{memory_value}    Create List
+        FOR    ${m_usage}    IN    @{memory}
+               ${m_value}    Convert To Integer    ${m_usage}[:-2]
+               Append To List    ${memory_value}    ${m_value}
+        END
+        Run Keyword And Continue On Failure   RHODS Performance Result Validation     ${memory_value}
```

Comment:
this might be another test step, out of this kw. 
`
1. Capture values
2. Validate Values
`

Reply:
this keyword is captuing memory data, isn't it? And in the same kw the validation runs. So, I'm proposing to bring the validation outside of the keyword which captures the data.

Reply:
talked offline and find a solution